Firestore/core/test/firebase/firestore/remote/CMakeLists.txt

```diff
@@ -27,3 +27,17 @@ cc_test(
     firebase_firestore_protos_libprotobuf
     firebase_firestore_remote
 )
+
+set_target_properties(firebase_firestore_remote_test
+  PROPERTIES
+    # Treat headers from dependencies as *non* system includes (i.e. use -I
+    # rather than -isystem.) Required, as otherwise libprotobuf includes
+    # imported via -isystem are examined last, allowing nanopb includes to take
+    # priority.
+    NO_SYSTEM_FROM_IMPORTED true
```

Comment:
This seems somewhat scary. Is there an equivalent required for the Xcode project?